db/migrate/20150709180535_create_albums.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAlbums < ActiveRecord::Migration
+  def change
+    create_table :albums do |t|
+      t.string  :title
+      t.string  :artist
+      t.string  :genre
+      t.integer :length
```

Comment:
these should probably all have `null: false` added to them, given the nature of your validations